src/screens/Library.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import ButtonComponent from "../components/Button";
+import Loader from "../components/Loader";
+import Card from "../components/Card";
+import { useFetch, API_Candidates } from "../utility";
+import Likes from "./Likes";
+import styles from "./Profile.module.css";
+import Tags from "../components/Tags";
+
+const Library = () => {
+  const [dataCandidate, loadingCandidate] = useFetch(API_Candidates);
+  const [display, setDisplay] = useState(true);
+
+  return (
+    <>
+      <div onClick={() => setDisplay(!display)}>
+        ;
+        <ButtonComponent size="small" variant="primary">
+          Toggle
+        </ButtonComponent>
+      </div>
+      {display ? (
+        loadingCandidate && display ? (
+          <Loader />
+        ) : (
+          dataCandidate.map((item) => (
+            <div className={styles.profile} key={item.id}>
+              <div className={styles.hero}>
```

Comment:
To keep the same format for cards on both `display` states, maybe you can use the same `Card` component with a few tweaks to its props?